Fix note regarding installation of Pipenv in Readme

Etone opened the pull request:
- Changed wording to represent current state
- Added link to Pipenv installation instructions

Fix: #22

Commit:
Fix note regarding Pipenv installation in Readme.md

Commit:
Fix typo

Michionlion commented on README.md:
```diff
@@ -23,10 +23,9 @@ from there. Additionally, [Gradle](https://gradle.org/) is required to actually
-NOTE: GatorGradle will automatically install [Pipenv](pipenv.readthedocs.io),
-but does not add the executable to `$PATH`. This can be done manually if the
-user wishes to use Pipenv for other projects.
+NOTE: GatorGradle will **NOT** automatically install [Pipenv](pipenv.readthedocs.io). Nothing other then GatorGradle will be installed automatically.
```
Quick correction: GatorGrader will be installed automatically.

Michionlion reviewed (changes requested):
Small changes must be made to pass Travis (the build errored due to network conditions the first run, so I've rerun it) -- `README.md` must pass `mdl` (see the log [here](https://travis-ci.org/GatorEducator/gatorgradle/builds/437010797#L904)).

Other than that, looks good! Thanks for the PR! :)

Commit:
Fix wording and pipenv reference link

- fixed wording regarding GatorGradle install and pipenv
- reference pipenv link with https:// so it will not search in file tree
- fixed mdl mistakes

Fix: #22

Commit:
Fix mdl for line length in line 27

Etone commented:
Sorry for the inconvenience, seems like i had my mdl plugin in vscode disabled. Should be fixed. While at it i fixed the reference to pipenv docs as they always linked to the local file tree.

Michionlion commented:
No problem, thanks for the quick turnaround!
